bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Add a final comment :</p><p>"Now you are a hero with Rototrans !"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>